Modules/IO/DCMTK/include/itkDCMTKSeriesFileNames.h

```diff
@@ -162,7 +162,7 @@ class ITKIODCMTK_EXPORT DCMTKSeriesFileNames : public ProcessObject
   void
   AddSeriesRestriction(const std::string & /* tag */)
   {
-    // m_SerieHelper->AddRestriction(tag);
+    // m_SeriesHelper->AddRestriction(tag);
```

Comment:
This might not be a misspelling. Does it compile when uncommented?

Reply:
grepping through the code, there is no actual variable m_SeriesHelper or m_SerieHelper.  Neither one exists, so I guess it's old, defunct code.